lang/en.yml

```diff
@@ -377,10 +377,10 @@ en:
     ONCOMPLETEMESSAGE: "Thanks, we've received your submission."
     ORSELECTAFIELDTOUSEASFROM: '.. or select a field to use as reply to address'
     ORSELECTAFIELDTOUSEASTO: '.. or select a field to use as the to address'
-    PLURALNAME: 'Base Pages'
+    PLURALNAME: 'User Defined Forms'
```

Comment:
This change is a correct change. The old name is just wrong and must have been manually edited in transifex.